### DISCUSSION TITLE
Fullscreen tigger full screen in browser

### DISCUSSION THREAD

**Discussion by Set27** [prasanthrangan/hyprdots#901] [Q&A]
Is there an option to not enter browser fullscreen mode using alt + enter?

**Comment by RookiexCookie** [comment]
you change the keybindings in `~/.config/hypr/keybindings.conf`  for the fullscreen

**Reply by Set27** [reply]
No matter what key binding is

**Comment by kRHYME7** [comment]
> Is there an option to not enter browser fullscreen mode using alt + enter?

No specific config in hyprland yet, but can be written as a script on top of the togglefullscreen.

**Reply by kRHYME7** [reply]
```

#! /bin/bash

DISPATCHER="fullscreen"
WINDOW="firefox"

initial_class=$(hyprctl activewindow -j | jq -r '.initialClass')

if [ "$initial_class" != "$WINDOW" ]; then
    hyprctl dispatch "$DISPATCHER"
fi

```

You can play with this script
